Dockerfile-rails

```diff
@@ -1,7 +1,7 @@
-FROM ruby:2.6.2
+FROM ruby:2.7.0
 
 RUN curl -sL https://deb.nodesource.com/setup_9.x | bash -
-RUN apt-get update -qq && apt-get install -y build-essential libpq-dev postgresql-client nodejs
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev postgresql-client npm
```

Comment:
[nits] I think it would be better to uninstall both nodejs and npm in L9

Reply:
done